hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_8_0/4813-fix-tag-nonunique.yaml

```
@@ -0,0 +1,4 @@
+---
+type: fix
+issue: 4813
+title: "A previous implementation caused tags to be non-unique. This could cause `NonUniqueResultException` errors in some cases. This lookup path has been fixed, and that error will no longer be thrown."
```

Recommended rewording for clarity:

```suggestion
title: "Under heavy concurrency, a bug resulted in identical tag definitions being rejected with a `NonUniqueResultException` some of the time.  This has been corrected."
```